### ISSUE TITLE
Add keyboard navigation

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/resistance-in-a-wire#87]
Like https://github.com/phetsims/ohms-law/issues/49, we would like to add keyboard navigation to this sim.

**Comment by jessegreenberg** [comment]
@mbarlow12 added keyboard navigation to this sim, here is the latest link:
http://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.4.0-dev.1/resistance-in-a-wire_en.html?accessibility

In the meeting on 6/20/17 we discussed adding a modifier key to make the slider move at a smaller precision.  We agreed on the "shift" modifier key.

**Comment by mbarlow12** [comment]
Completed shift key step modification 15b5d955a5b60112adf91b12302f3e3b5fd53cd1

**Comment by jessegreenberg** [comment]
@emily-phet @arouinfar @amanda-phet @amyh-phet in 6/27/17 meeting, we discussed that the keyboard steps could be larger for some sliders since the shift modifier will give user fine control. If they should change, what should they be?

@emily-phet what is the next step for the keyboard help dialog in this sim?

**Comment by emily-phet** [comment]
@jessegreenberg Something to note regarding slider stepsize. When using my Mac and the fn modifier key to increase the slider stepsize - the stepsize changes at the far end of the slider to allow access to the min/max of the slider range. So either a) this is functionality we can make use of, or at the very least, b) it's not unusual behavior for stepsize to change during interaction (from a usability perspective, we don't have to keep the same stepsize for the entire slider range).

**Comment by jessegreenberg** [comment]
@emily-phet, here is a version with the shift key acting like a modifier key for fine movement:

http://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.4.0-dev.2/resistance-in-a-wire_en.html?ea&brand=phet&accessibility

**Comment by emily-phet** [comment]
@jessegreenberg I've decided for keyboard nav that it is important to have a moderate step value by default, and friendly numerical values - even if this means the step isn't equal across the span of the slider.
- Please have the voltage slider default to a 0.5V increment (from the bottom that would look like 0.1, 0.5, 1.0, 1.5,...9.0). Also add the ability for the shift modifier key to allow increments of 0.1V. Let me know if you need to investigate how to allow changes in incement across the range of a slider, or if you already have an idea of how to do this.
- For Resistance, have the slider default to a 20 ohms. Also add the ability for the shift modifier key to allow increments of 1 ohm.

**Comment by jessegreenberg** [comment]
Sounds great @emily-phet, reassigning to @jessegreenberg and @mbarlow12.

**Comment by jessegreenberg** [comment]
@emily-phet here is an updated version:

http://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.3.0-dev.2/resistance-in-a-wire_en.html?accessibility

Can you please review?

**Comment by emily-phet** [comment]
@jessegreenberg Looks great!

**Comment by jessegreenberg** [comment]
Excellent!  @emily-phet what are the next steps before deploying a new version with this?  In my mind, they are the keyboard help dialog, dev testing, then RC testing.  Is there anything else?

**Comment by emily-phet** [comment]
@jessegreenberg Keyboard help dialog is in progress, as well as sonification for UI components. It's too early to know for sure if we'll be able to get sonification in within a reasonable timeline, but it's possible so let's hold off on at least RC testing (maybe dev testing as well?) until I have a better sense of the sonification plan. I should be able to make a call on whether or not to proceed with publication without sonification within the next two weeks.

**Comment by jessegreenberg** [comment]
It looks like the navigation order was slightly changed in the investigation of #28.

**Comment by jessegreenberg** [comment]
Regarding  https://github.com/phetsims/resistance-in-a-wire/issues/87#issuecomment-314900304, we decided that this sim can proceed with deployment for keyboard navigation. In #101 we added a keyboard help dialog that will work until we settle on a long term design. Keyboard nav is complete so we can close this issue.
